content/reference/v1beta1/repeatable.md

```diff
@@ -0,0 +1,278 @@
+---
+date: 2021-07-13
+linktitle: "Repeatable"
+title: Repeatable Config
+description: "Repeatable config generates copies of kubernetes manifest items."
+weight: 9
+---
+
+## Repeatable Items
+
+A repeatable config item copies a YAML array entry or YAML document for as many values as are provided. Any number of values can be added to a repeatable item to generate additional copies.
+
+**Repeatable Items presently only work for text and textarea types**
+
+To make an item repeatable, set `repeatable` to true
+```yaml
+    - name: ports
```

Comment:
I did this to match the Config page, where yaml snippets from the middle of a file are indented.  Since this is every case on the page I think it's reasonable to eliminate the indentation, but we should consider that it will look differently than the Config page.

Let me know what is preferable and I'll adjust it

Reply:
I figured maybe that was the reason, but then I noticed that some complete manifests are indented too, like in the examples at the bottom of the page. I think it's fine/good to indent the snippets from the middle of a file, so let's just fix the complete file indentations.

Reply:
Thanks for catching those at the bottom, totally slipped my radar!  I fixed the indenting on the full yaml manifests if you want to double check.